examples/1_clap_for_everything.py

```diff
@@ -10,17 +10,18 @@
 import logging
 import os
 
-from pykeybasebot import Bot, ContentType
+import pykeybasebot.types.chat1 as chat1
+from pykeybasebot import Bot
 
 logging.basicConfig(level=logging.DEBUG)
 
 
 async def handler(bot, event):
-    if event.msg.content.type != ContentType.TEXT:
+    if event.msg.content.type_name != chat1.MessageTypeStrings.TEXT.value:
```

Comment:
`chat1.MessageTypeStrings.TEXT.value` <-- this is pretty unfortunate for something as common as this '__'